arb_os/arbowner.mini

```diff
@@ -75,6 +101,144 @@ impure func arbowner_giveOwnership(topFrame: EvmCallFrame, calldata: ByteArray)
     evmOp_return(0, 0);
 }
 
+impure func arbowner_addToReserveFunds(topFrame: EvmCallFrame, calldata: ByteArray) { // () payable
+    if (bytearray_size(calldata) != 4) {
+        evmOp_revert_knownCodePc(address(const::Address_ArbOwner), 50, 0, 0);
+    }
+    let callvalue = evmCallFrame_getCallvalue(topFrame);
+    if (callvalue > 0) {
+        gasAccounting_addToReserveFunds(callvalue);
+    }
+    evmOp_return(0, 0);
+}
+
+impure func arbowner_getFeeRecipient(topFrame: EvmCallFrame, calldata: ByteArray) { // () -> (address)
```

Comment:
We should consider making this publicly callable

Reply:
Same as above.